### ISSUE TITLE
Additional sources / visualizations should be investigated from Genome Nexus

### ISSUE THREAD

**Issue by kkrysiak** [genome/civic-client#1628]
Consider pulling in data from cancerhotspots.org. Comparison of other options not available through myvariant.info:
https://docs.google.com/spreadsheets/d/1xML949NWzJGcvltjlquwSRIv79o13C_sfrVPAU5ci9Q/edit#gid=258442188
- Although this makes me think that Cancer Gene Census through open-CRAVAT would be another good resource

Visualizations, including loliplots:
https://docs.genomenexus.org/tools

**Comment by malachig** [comment]
Yeah, Open-CRAVAT integration would be great, has many of these annotations, and has been proposed here: https://github.com/griffithlab/civic-client/issues/1626
